LauncherInjector/main.cpp

```diff
@@ -224,6 +227,17 @@ bool LoadNorthstar()
 	}
 
 	((bool (*)())Hook_Init)();
+	if (!noLoadPlugins)
+	{
+		Hook_Init = GetProcAddress(hHookModule, "LoadPlugins");
+		if (!hHookModule || Hook_Init == nullptr)
+		{
+			printf("Failed to get function pointer to LoadPlugins of Northstar.dll\n");
+			return false;
```

Comment:
Maybe print the error message? See my stub code for how to do this without all the junk in `LibraryLoadError`.